packages/expo-camera/src/next/CameraView.tsx

```diff
@@ -151,14 +152,26 @@ export default class CameraView extends React.Component<CameraProps> {
   }
 
   /**
-   * Dimiss `DataScannerViewController`
+   * Dimiss the scanner presented by `launchScanner`.
+   * @platform ios
    */
   static async dismissScanner(): Promise<void> {
     if (Platform.OS === 'ios' && CameraView.isModernBarcodeScannerAvailable) {
       await CameraManager.dismissScanner();
     }
   }
 
+  /**
+   *
+   * Callback that is invoked when a bar code has been successfully scanned. The callback is provided with
+   * an object of the `ModernBarcodeScanningResult` shape, where the `type`
+   * refers to the bar code type that was scanned and the `data` is the information encoded in the bar code
+   * (in this case of QR codes, this is often a URL). See `BarCodeType` for supported values.
```

Comment:
```suggestion
   * (in this case of QR codes, this is often a URL). See [`BarCodeType`](#barcodetype) for supported values.
```

Let's add the link here to BarCodeType reference.